src/pages/billing/List.ts

```diff
@@ -0,0 +1,37 @@
+import { div, h } from 'react-hyperscript-helpers';
+import { ListPage } from 'src/billing/List/ListPage';
+import FooterWrapper from 'src/components/FooterWrapper';
+import TopBar from 'src/components/TopBar';
+import * as Nav from 'src/libs/nav';
+import * as Style from 'src/libs/style';
+
+interface ListProps {
+  queryParams: {
+    selectedName: string | undefined;
+  };
+}
+
+export const List = (props: ListProps) => {
```

Comment:
Then convention I think we want is to call these XxxxPage like EnvironmentsPage ListPage... tho BillingListPage feels better here.... and have the filename match.

Reply:
Done in 715a7363a4a302731ae1b4f719eede1ccdd846d4